sig-platform-dev/README.md

```diff
@@ -0,0 +1,83 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+-->
+# Platform Dev Special Interest Group
+
+SIG Platform Dev (SIG-PD) is a cross-cutting SIG whose mission is to design, build, and maintain a cohesive, first-party Platform Development Kit (PDK) for developers who want to extend the Kubernetes platform by adding new APIs or enhancing existing APIs, as well as to advocate on behalf of such developers by driving feedback into other SIGs.
+
+## Meetings
+* Regular SIG Meeting: [Tuesdays at 10:00 PT (Pacific Time)]() (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
+
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+
+* Jimmy Zelinskie (**[@jzelinskie](https://github.com/jzelinskie)**), Red Hat
+* Anthony Yeh (**[@enisoc](https://github.com/enisoc)**), Google
+
+### Technical Leads
+The Technical Leads of the SIG establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Jimmy Zelinskie (**[@jzelinskie](https://github.com/jzelinskie)**), Red Hat
+* Anthony Yeh (**[@enisoc](https://github.com/enisoc)**), Google
+* Phillip Wittrock (**[@pwittrock](https://github.com/pwittrock)**), Google
+
+## Contact
+* [Slack](https://kubernetes.slack.com/messages/sig-platform-dev)
+* [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-platform-dev)
+* [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fplatform-dev)
+
+## Subprojects
+
+The following subprojects are owned by sig-platform-dev:
```

Comment:
I lumped kubebuilder in with the "experimental subprojects" section in the charter because there were claims on the original thread that it and metacontroller are out of scope for the Kubernetes project. I disagree on both counts, but in light of the concerns raised, I decided to decouple that decision from the founding of SIG-PD and leave it up to the KEP process to decide later.

Reply:
That is fine.

Reply:
That is fine for now.